ikarus/finiteelements/mechanics/linearelastic.hh

```diff
@@ -135,10 +140,7 @@ public:
    * \return The material tangent matrix.
    */
   auto materialTangent() const {
-    if constexpr (myDim == 2)
-      return planeStressLinearElasticMaterialTangent(mat_.emodul, mat_.nu);
-    else if constexpr (myDim == 3)
-      return linearElasticMaterialTangent3D(mat_.emodul, mat_.nu);
+    return mat_.template tangentModuli<StrainTags::linear, true>(Eigen::Vector<double, 6>::Zero());
```

Comment:
That `materialTangent` does not receive strains might by a problem if we have a geometrically linear problem but materially non-linear and we want to write stress to file.

Reply:
But this is not in the scope of this PR.

Reply:
yeah passing in 6 zeros felt a bit hacky, but we can worry about that when we have such materials.

Reply:
(Can you write a short comment on that above the function call?) I added a commit I hope that does not break anything